### ISSUE TITLE
Missing Plugins in package.xml

### ISSUE THREAD

**Issue by db-ks** [opentap/OpenTap.Python#130]
@alan-copeland-keysight is building an OpenTAP package that contains OpenTAP plugins written in Python. In the built *.TapPackage the file package.xml does not contain the plugins in `<Plugins>` section:

```xml
   <Plugins>
        <Plugin ...></<Plugin>
        <Plugin ...></<Plugin>
        <Plugin ...></<Plugin>
        ...
   </Plugins>
```

**Comment by alan-copeland-keysight** [comment]
This appears to be a problem for all Python-based plugins. PythonExamples, for example, does not show any plugins information in the package.xml or on packages.opentap.io

![image](https://github.com/opentap/OpenTap.Python/assets/69871845/895807c1-2356-4d75-b113-0b6177bb1759)
![image](https://github.com/opentap/OpenTap.Python/assets/69871845/8132b227-e7b0-4e9c-899d-019acf4c3b53)

**Comment by alan-copeland-keysight** [comment]
@rmadsen-ks - is this a problem with **tap package**, or the Python plugin?

**Comment by rmadsen-ks** [comment]
Hi @alan-copeland-keysight,

I think this feature has not been considered that important until now. 

Basically, OpenTap.Package only supports doing this for .net assemblies. So I guess we have to add support in the python plugin?

Maybe we can add it with some serializer plugin, but it is going to be a bit hacky.

**Comment by alan-copeland-keysight** [comment]
Without it, KS8500 is unable to add Python packages. The workaround is equally hacky: add a C# ResultListener that does nothing, but allows the plugin to be "discovered".

**Comment by rmadsen-ks** [comment]
If you can work with that for now, then maybe we can solve the problem properly in OpenTAP. 

Theoretically I think the Python plugin does everything it should in respect to this now.

**Comment by rmadsen-ks** [comment]
This is fixed in 3.2.
